locations/locations_pop_er.json

```diff
@@ -7388,7 +7388,8 @@
               {
                 "name": "DJ Khaled - Let's go Golfing!",
                 "access_rules": [
-                  "sword"
+                  "sword",
+                  "[glitches]"
                 ],
```

Comment:
I would change this to 
```json
"access_rules": [
  "[sword]"
]
```

since it's not really "glitchy" to do the skulls without the sword. Also makes it more consistent with how the other "weapon-optional areas" are implemented.

Reply:
So wait, what do the brackets mean then? Just like, "if you don't have the item in the brackets, but you have everything else required, it will show up as yellow"?

Reply:
pretty much exactly that.